runDocker.ts

```diff
@@ -49,5 +39,52 @@ async function launchDockerSearch() {
   const stream = await container.attach({ stream: true, stderr: true })
   stream.pipe(process.stderr)
   await container.start()
-  return container
+
+  const signals = ['message', 'SIGTERM', 'SIGINT']
+  signals.forEach((signal) => {
+    process.on(signal, async () => {
+      await container.kill()
+    })
+  })
+
+  await container.wait()
+}
+
+export const launchDocker: SearchEngineLauncherFunction = async ({
+  dataDir,
+  logsDir,
+  engine,
+  port,
+  options,
+}) => {
+  const argv = {
+    dataDir,
+    logsDir,
+    engine,
+    port,
+    options,
+  }
+  // FIXME: fork accepts either a string or URL as the first argument. @types/node has defined the modulePath type as string only.
```

Comment:
[the underlying issue with the types library has been fixed](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69819/files)

Reply:
Fixed.